code/style.css

```diff
@@ -1,7 +1,410 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+html, body {
```

Comment:
Your content is a bit too wide on mobile:
<img width="393" alt="Skärmavbild 2021-02-09 kl  12 21 24" src="https://user-images.githubusercontent.com/51529201/107356634-54e5fd80-6ad1-11eb-84bd-661e224bf4ab.png">

and on tablet I would suggest giving the video 100% height instead of 100% width:
<img width="616" alt="Skärmavbild 2021-02-09 kl  12 22 55" src="https://user-images.githubusercontent.com/51529201/107356843-9b3b5c80-6ad1-11eb-863f-b7731a492e6e.png">

Reply:
I rebuilt the css completely to work from mobile first (I had originally started with desktop and it proved problematic) it should be way more responsive and smooth now. 😃